Sources/Firework/APIRequest.swift

```diff
@@ -23,6 +26,10 @@ public extension APIRequest {
     var queryItems: [URLQueryItem]? { nil }
     var acceptableStatusCodes: Range<Int> { 200..<400 }
     
+    var acceptableContentTypes: [String] {
+        headers?["Accept"]?.components(separatedBy: ",") ?? ["*/*"]
```

Comment:
[note]
The default value is the same as Alamofire.
https://github.com/Alamofire/Alamofire/blob/78424be314842833c04bc3bef5b72e85fff99204/Source/Validation.swift#L71-L77